woocommerce/woocommerce-android
# HACK week: AI Thank you note

### Description


Adds a "Thank-you note" generator utilizing Jetpack AI.

This feature is available on Orders with "Completed" status. A generated note can then be copy/pasted, or shared using the Android share sheet.

As this is a HACK week work, the design/wording will likely need further iteration. For now please focus on functionality and code for the review.

Check video below for demo:

[ai-thank-you-note-demo.webm](https://github.com/woocommerce/woocommerce-android/assets/266376/4611d77f-18f7-42e0-8d16-3f5b6afd8616)

### Testing instructions


1. Use a WordPress.com site,
2. Find an Order with "Completed" status, open it,
3. Tap the "Create thank you note" button,
4. Wait and ensure a thank-you note is generated,
5. Try also the copy, feedback, "Regenerate" and "Share" option. (For feedback you will need to check in Logcat whether it's logged properly or not).
6. Also try turning off internet while Regenerating to test error UI.
